moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -312,6 +312,12 @@ export const useColumns = ({ hiddenColumns }) => {
         },
         width: COLUMN_WIDTHS.xlarge,
       },
+      {
+        headerName: "Substantial completion date",
+        field: "substantial_completion_date",
+        valueFormatter: (value) => value && formatDateType(value),
+        width: COLUMN_WIDTHS.small,
```

Comment:
Part of me feels like this could be updated to `COLUMN_WIDTHS.medium` so that the entire column header is visible by default. Counter point is that we don't do this for other longer column headers like **Public process status** or **Interim MPD (Access) ID**.

What you you think?

Reply:
i had that thought as well! since this is a requested feature and i think will be highly utilized i think it's fair to give it better visibility. i think it looks better at medium so i will push that but i am open to changing ti back!